apps/src/templates/sectionAssessments/MatchDetailsDialog.jsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable react/no-danger */
```

Comment:
this doesn't look needed. remove from here, plus wherever you copied it from?

Reply:
Will do! Going to fix on the part 4 branch and ship this PR